### PR TITLE
 Added a hyperlink in README.md

### PR DESCRIPTION
**Pull request by NishcheyBhutani**
I added a hyperlink to->'Knowledge of git & github.' present in '3.Skill Set' section in README.md ,clicking it will take the user to code-for-cause's awesome youtube tutorial on git/github from where it could be learned in just 15 minutes.

**Issue Number**

fixes #

**Describe the changes you've made**

**Describe if there is any unusual behavior (Any Warning) of your code(Write `NA` if there isn't)**

**Additional context (OPTIONAL)**

**Test plan (OPTIONAL)**

A good test plan should give instructions that someone else can easily follow.

**Checklist**

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

**Review by Siddhant-K-code — CHANGES_REQUESTED** [review]
Resolve files conflict.

**Comment by NishcheyBhutani** [comment]
> Resolve files conflict.

i have resolved the conflicts,can u pls check it now!?

**Comment by NishcheyBhutani** [comment]
@kunal-kushwaha @KeenWarrior @Abhishek-kumar09 can u pls check and merge!?

File: README.md [hunk 1]
```diff
@@ -38,12 +38,153 @@ Thank you for considering and taking the time to contribute! Before contributing
...
+1. Now you have your own copy of project. Here you have to start your work.
+2. Go to desired location on your computer where you want to set-up the project.
+3. Right click there and click on `git bash`. A terminal window will pop up
+4. Type the command `git clone <your-fork-url>.git` and hit enter.(Refer the image to copy url)
+5. Wait for few seconds till the project gets copied
+   <img src="./doc/images/copy.png" alt="Demo copy image" align="center"/>
+
+### 🚩 Setting up remote :
+
+1. Now you have to set up remote repositories
```
Comment (Abhishek-kumar09) [inline README.md:67]: These things are already in contributing.md https://github.com/codeforcauseorg/edu-client/blob/main/CONTRIBUTING.md

We may not need the full contrubution guideline here.

Comment (NishcheyBhutani) [reply README.md]: @Abhishek-kumar09 can u pls check it now!?

File: README.md [hunk 2]
```diff
@@ -38,11 +38,11 @@ Thank you for considering and taking the time to contribute! Before contributing
 
 ### ⚡ 3. Skill set :
 
-1. Knowledge of git & github.
+1. [Knowledge of git & github.](https://www.youtube.com/watch?v=4iDQ7CHHZZM)
 2. JavaScript
 3. [ReactJS](https://reactjs.org/)
 4. Mobile development Experience in Any Tech (optionally)
-5. [Ionic](https://ionicframework.com/) / [Capacitor](https://capacitorjs.com/) (optionally)
+5. [Ionic / Capacitor](https://capacitorjs.com/) (optionally)
```
Comment (Abhishek-kumar09) [inline README.md:45]: Ionic and capacitor are two different things.

File: README.md [hunk 3]
```diff
@@ -38,11 +38,11 @@ Thank you for considering and taking the time to contribute! Before contributing
 
 ### ⚡ 3. Skill set :
 
-1. Knowledge of git & github.
+1. [Knowledge of git & github.](https://www.youtube.com/watch?v=4iDQ7CHHZZM)
```
Comment (Abhishek-kumar09) [inline README.md:41]: We may not need this since we want to make docs promotion free without any third party videos but only official documentation

**Comment by Abhishek-kumar09** [comment]
This is not needed more.

Please feel free to open the PR if you have anything else to add.
I am closing it for managing the other open Pull requests.

**Pull request closed without merging**